### PR TITLE
5595 & 6144 - Icon puppeteer test for new icons

### LINKED ISSUE #5854: Icons: Create a Puppeteer Script for change Launch Icon
Describe the Issue
Create a unit testing for change Launch Icon.

See attached BDD for more information.
[BDDStory_github5595.docx](https://github.com/infor-design/enterprise/files/7520826/BDDStory_github5595.docx)

Related github issue
#5595

### LINKED ISSUE #6199: Unit Testing: Create a Puppeteer Script for new Mobile Icon
Describe the Issue
Create a unit testing for Add new Mobile Icon

See attached BDD for more information.
BDDStory_github#6144[85].docx

Related github issue
#6144

### PR DESCRIPTION
**Pull request by jbrcna**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

**Related github/jira issue (required)**:
closes #5854 , related to #5595 
closes #6199 , related to #6144 

**Steps necessary to review your pull request (required)**:

1. pull this branch
2. build and run the app
3. run npm run e2e:puppeteer icon
4. It should pass.

Result should be

`Mobile icon`
image

`Launch icon`
_the image below shows the diff_
image

**Included in this Pull Request**:
- [ ] An e2e or functional test for the bug or feature.
- [ ] A note to the change log.

### CODE REVIEW

**Review by tmcconechy — CHANGES_REQUESTED** [review]
I think you need to push the new image as well for the test?

**Comment by ericangeles** [comment]
If this is now open for review, please make your PR readable for the reviewers. You can check other PRs for references.

**Comment by ericangeles** [comment]
Can you add the changelog for the two tickets? Once added, I'll approve this PR.

**Review by tmcconechy — DISMISSED** [review]

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 ## v4.62.0 Features
 
 - `[Spinbox]` Create a puppeteer script for Spinbox Field sizes on mobile. ([#6197](https://github.com/infor-design/enterprise/pull/6197))
+- `[Icon]` Create a puppeteer script for new icons. ([#6202](https://github.com/infor-design/enterprise/pull/6202))
```
Comment (ericangeles) [inline docs/CHANGELOG.md:6]: Make it in alphabetical order, then good to go.

**Comment by jbrcna** [comment]
> Can you add the changelog for the two tickets? Once added, I'll approve this PR.

change log added 👍

File: docs/CHANGELOG.md (hunk 1 above)
Comment (ericangeles) [reply docs/CHANGELOG.md:5]: The ticket numbers and the links should be the ticket issues, not the PR.

- `[Icon]` Create a puppeteer script for the new launch icon. ([#5854](https://github.com/infor-design/enterprise/issues/5854))
- `[Icon]` Create a puppeteer script for the new mobile icon. ([#6199](https://github.com/infor-design/enterprise/issues/6199))

**Review by ericangeles — CHANGES_REQUESTED** [review]

**Comment by jbrcna** [comment]
change log updated. and change Create to Created as discussed in our chat. :grin:

**Review by tmcconechy — APPROVED** [review]

**Review by ericangeles — APPROVED** [review]

**Pull request merged**